libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -230,11 +230,6 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 		db.setLockingEnabled(false);
 	}
 
-	@Override
-	public void onOpen(SQLiteDatabase db) {
-		(new SmartStore(db)).resumeLongOperations();
```

Comment:
That could not work since no encryption key was passed in.
Moving that code to SmartStore.java.
I also need to add a test exercising that code path.

Reply:
Will the tests be a followup PR?  Or can they be included in this one?

Reply:
I'll add a test today to the PR.

Reply:
@gkotula there were tests already for the long operations stuff but they were not realistically simulating a restart. I addressed that in https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2352/commits/d289d816a0aaa27746ebdc54e70d2082709c3899